docs/appconfig/schema-definitions-featureflagsconfig-properties.md

```diff
@@ -0,0 +1,16 @@
+# Untitled undefined type in AppConfig Schema
```

Comment:
this looks weird

Reply:
I don't think we actually use these files - but I will take a short look